app/components/application_form/autocompleter_field.rb

```diff
@@ -17,10 +19,16 @@ class AutocompleterField < Superform::Rails::Components::Input
       :clade, :region
     ].freeze
 
+    slot :append
+
+    # Make slot accessor public (Phlex::Slotable makes them private by default)
+    public :append_slot
+
```

Comment:
@mo-nathan The reason for this change is that in Phlex, a `slot` rather than a `prop` is the best way to handle a "component arg" that contains HTML. If it's a `slot` it's got to be valid HTML; if it's a `prop` it's typed as a `String` but unvalidated as HTML.

The `:append` slot is for HTML that may be printed below the autocompleter field.